### ISSUE TITLE
No way to remove a Pen from a Collection

### ISSUE THREAD

**Issue by kevingimbel** [CodePen/redesign#164]
It seems like there's no way to remove a Pen from a collection. I just added one by mistake but on the Collection View I can't find a button to remove it now.

The same happens with my own pens, I can delete them and add them to other collections but I see no way of removing them from a specific collection. 

Otherwise, great redesign! Feels modern and "good" - really like it! :smiley: 

![collection_view](https://cloud.githubusercontent.com/assets/2161714/10241069/55788ba4-68e4-11e5-9005-45f9e47e9baa.png)

**Comment by oliverholretz** [comment]
you can click edit collection in the top right menu(the menu under the blue date), then you can hover over a pen so the hover panel comes up, there should be a button to delete there.
![screenshot-codepen](https://cloud.githubusercontent.com/assets/10248190/10245179/856761b6-6905-11e5-870a-a26ba5503350.png)

**Comment by jeremy-jameson** [comment]
Unlike Kevin, I actually saw the Delete button (as illustrated in Oliver's screenshot). However, when I clicked "Delete", it deleted the pen completely (rather than removing it from the collection -- which is what I expected).

Granted, it did prompt me to confirm the pen would be deleted entirely -- and I did click the "I Understand, Delete My Pen" button. Nevertheless, I really just wanted to remove the pen from the collection.

In my case, fortunately, I was working with a simple template that only took a couple of minutes to recreate (which is why I wasn't too worried about deleting the pen).

**Comment by oliverholretz** [comment]
i didn't actually see that it deletes the pen, i only assumed that it was a "delete from collection".
The edit collection didn't change anything, i just thought i had clicked it a few seconds before that, but no popup came up those times i did

**Comment by oliverholretz** [comment]
i didn't actually see that it deletes the pen, i only assumed that it was a "delete from collection".
The edit collection didn't change anything, i just thought i had clicked it a few seconds before that, but no popup came up those times i did

**Comment by chriscoyier** [comment]
This is a complete oversight on our part. We'll get a button to remove it from the collection in there soon.

**Comment by chriscoyier** [comment]
This is a complete oversight on our part. We'll get a button to remove it from the collection in there soon.

**Comment by kevingimbel** [comment]
@oliverholretz I saw the delete button (see the quote from my original post)
> The same happens with my own pens, I can delete them and add them to other collections but I see no way of removing them from a specific collection.

But as @jeremy-jameson pointed out this "x Delete" Button will delete the pen instead of remove it. In my case I needed to remove a pen from another person from my collection. Clicking "Edit Collection" only brings up the Information Modal with the Title and Description. 

Anyway, as @chriscoyier pointed out this options seems to be missing entirely. Nothing too bad though, when redesigning a whole application this can happen so I'll just wait for the button to return. :smile:
